src/Conf.ml

```diff
@@ -642,6 +643,17 @@ module Formatting = struct
       (fun conf x -> {conf with doc_comments= x})
       (fun conf -> conf.doc_comments)
 
+  let end_of_line_in =
```

Comment:
Something like `break-before-in` would still be more consistent with the existing option naming scheme.